package.json

```
@@ -19,6 +19,8 @@
     "garbage_collector": "node extensions/gc/service.js",
     "test": "mocha --recursive tests/unit --timeout 30000",
     "cover:test": "nyc --clean --silent yarn run test && nyc report --report-dir ./coverage/test --reporter=lcov",
+    "e2e": "mocha --recursive $(find tests/e2e -name '*.js') --timeout 30000",
+    "cover:e2e": "nyc --clean --silent yarn run e2e && nyc report --report-dir ./coverage/e2e --reporter=lcov",
```

to avoid duplication, we may generate a [generic `cover` script](https://github.com/scality/cloudserver/blob/development/8.8/package.json), something like: 

```suggestion
    "e2e": "mocha --recursive $(find tests/e2e -name '*.js') --timeout 30000",
    "cover": "nyc --clean --silent yarn run",
    "postcover": "nyc report --report-dir ./coverage/test --reporter=lcov",
```

which can be run with `yarn run cover test` and `yarn run cover e2e`.
(but we would need to find a way to get the step name in the middle of the `postcover` command, so we can put the coverage in separate folder)